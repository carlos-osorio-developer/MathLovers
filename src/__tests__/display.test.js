import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../components/display';

it('renders correctly', () => {
  const tree = renderer.create(
    <Display total="473" next="11" operation="/" />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});