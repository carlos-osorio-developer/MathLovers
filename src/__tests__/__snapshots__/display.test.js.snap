// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="display"
>
  <p>
    473 / 11
  </p>
</div>
`;
